scripts/install_prerequisites.sh

```diff
@@ -35,16 +35,26 @@
 toolchain=''
 exclude_prereq=''
 install_all=true
+keep_sources=false
+tpls_dir="/opt/cudaq/tpls"
```

Comment:
rather than hardcode this, it would probably be good to look if CUDAQ_INSTALL_PREFIX is defined.